api/Navigator.json

```diff
@@ -2365,6 +2365,112 @@
           }
         }
       },
+      "unregisterProtocolHandler": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "38"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": [
+              {
+                "version_added": "25"
+              },
+              {
+                "version_added": "≤12.1",
+                "version_removed": "15"
+              }
+            ],
+            "opera_android": [
```

Comment:
I'm pretty sure the `registerProtocolHandler` for Opera for Android is wrong, can you fix it?

Reply:
I've tested it in real Opera for Android and it's not supported. I'll update this.